content/tutorial/tutorial.md

```diff
@@ -12,125 +12,130 @@ redirect_from:
   - "docs/tutorial-zh-CN.html"
 ---
 
-This tutorial doesn't assume any existing React knowledge.
+Deze tutorial vereist geen voorbestaande React kennis.
 
-## Before We Start the Tutorial {#before-we-start-the-tutorial}
+## Voordat we beginnen {#voordat-we-beginnen}
 
-We will build a small game during this tutorial. **You might be tempted to skip it because you're not building games -- but give it a chance.** The techniques you'll learn in the tutorial are fundamental to building any React apps, and mastering it will give you a deep understanding of React.
+
+We zullen een klein spelletje bouwen in deze tutorial. **Misschien ben je geneigd om dit over te slaan aangezien je geen games maakt -- maar probeer het.** De technieken die je zal leren in deze tutorial zijn fundamenteel om React applicaties te bouwen, eens je deze onder de knie krijgt, heb je een diep inzicht in React. 
```

Comment:
Replacing `tutorial` with `inleiding` for consistency. Also suggesting `maar geef het een kans` to stick closer to the original intention and have a better flowing sentence.

I do like the usage of `eens` but we could possibly simplify to `wanneer` (omitted in the suggestion for your discretion).

```suggestion
We zullen een klein spelletje bouwen in deze inleiding. **Misschien ben je geneigd om dit over te slaan aangezien je geen games maakt -- maar geef het een kans.** De technieken die je zal leren in deze inleiding zijn fundamenteel om React applicaties te bouwen, eens je deze onder de knie krijgt, heb je een diep inzicht in React. 
```